### PR TITLE
Updated the nix process

### PR DESCRIPTION
**Pull request by zo-el**

### CODE REVIEW

File: default.nix [hunk 1]
```diff
@@ -89,28 +89,75 @@ in
...
+
+    installPhase = ''
+      mv hp-admin $out
+    '';
+
+    checkPhase = ''
+      npm run test:ci
+    '';
+
+    doCheck = false;
```
Comment (samrose) [inline default.nix:114]:
```suggestion
    doCheck = false;
    meta.platforms = stdenv.lib.platforms.linux;
```

File: default.nix [hunk 2]
```diff
@@ -89,28 +89,75 @@ in
...
+      mv holofuel $out
     '';
 
     checkPhase = ''
-      npm run test:all
+      npm run test:ci
     '';
 
-    doCheck = true;
+    doCheck = false;
```
Comment (samrose) [inline default.nix:161]:
```suggestion
    doCheck = false;
    meta.platforms = stdenv.lib.platforms.linux;
```

**Review by samrose — CHANGES_REQUESTED** [review]

File: default.nix [hunk 3]
```diff
@@ -89,28 +89,77 @@ in
     buildInputs = [
       run-unit-test
       build-hp-admin
```
Comment (peeech) [inline default.nix:91]: I am not sure what is the purpose of this import here. `build-hp-admin` is essentially a shell script that installs npm modules and runs the `build` command. Same functions are encoded in `preConfigure` and `buildPhase` below. Would love to hear an opinion from @samrose or @yegortimoshenko on this.

File: default.nix [hunk 4]
```diff
@@ -89,28 +89,77 @@ in
...
+      libjpeg
+      libpng
+      libuuid
+      pango
+      pixman
+    ];
+
+    buildInputs = [
+      run-unit-test
+      build-holofuel
```
Comment (peeech) [inline default.nix:138]: Same comment here re: is this build phase necessary

File: default.nix (hunk 3 above)
Comment (zo-el) [reply default.nix:91]: @peeech This was just added for the users of the repo(hAppy team) to build their project using nix.

Comment (peeech) [reply default.nix:91]: ok, thanks for clarification

File: default.nix (hunk 4 above)
Comment (peeech) [reply default.nix:138]: resolved

**Comment by pjkundert** [comment]
Looks good to me.  If we want to use it against current envoy, we'd need to add an artifact from hclient (as per: https://github.com/Holo-Host/holo-nixpkgs/blob/201909/demo-cherry-pick/overlays/holo-nixpkgs/holofuel-app/default.nix), but we're working toward supporting the *new* "chaperone" SDK, so this is probably not necessary.

It looks good!

**Review by pjkundert — APPROVED** [review]
This is sufficient to get the `holofule-app` UI into the flow; we'll need to update it to a more recent branch, later, as this one doesn't work (even for the old hClient / envoy), due to needing an updated hClient.  That's OK for now, though.

**Pull request merged**